### PR TITLE
fix #28022 metrolyrics.com

### LINKED ISSUE #28022: metrolyrics.com
### Issue URL (Ads)
[http://www.metrolyrics.com/dig-down-lyrics-muse.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.metrolyrics.com%2Fdig-down-lyrics-muse.html)
### Comment
Username: @pedrogarcia
> Huge banner ads. Also, I could only successfully upload images to the adguard issue tracker after disabling adguard. 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | And 9
AdGuard version: | 2.12.247
Browser: | Chrome
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | enabled
Filters: | AdGuard Mobile Ads,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Spyware,<br>AdGuard Social Media,<br>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/28022

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -105,6 +105,7 @@ nationalfm.ro###box-fb-like
 kizlarsoruyor.com###contentpopup-container
 eva.ru###subscribe-dialog
 navalny.com###pushPopup
+metrolyrics.com###videoContainer
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/popups.txt]: Please limit rule to mobile devices only. The video is autostarting on Windows Chrome and when it is hidden, the user has no way to pause the video.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -43,6 +43,7 @@ vg247.com##div[id^="mobile-mpu-"]
 woman.ru##.mob-in__container
 woman.ru##.notification__ad
 woman.ru##div[data-name="MobListVideo"]
+metrolyrics.com###videoContainer
```
Comment (BlazDT) [inline MobileFilter/sections/specific_web.txt]: I meant with this kind, so it looks like this in Annoyances filter:
!+ PLATFORM(android, ios, ext_android_cb)
metrolyrics.com###videoContainer

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/specific_web.txt (hunk 2 above)
Comment (zloyden) [reply MobileFilter/sections/specific_web.txt]: I couldn't find this string in Annoyances filter, just !+ PLATFORM(ios, ext_android_cb). 
Can you show where it should be?

Comment (BlazDT) [reply MobileFilter/sections/specific_web.txt]: When you add !+ PLATFORM(android, ios, ext_android_cb) one line above the rule, it tells the compiler to only put the rule into filter lists for the mentioned platforms.

So it should like this in Annoyances filter:

eva.ru###subscribe-dialog
navalny.com###pushPopup
!+ PLATFORM(android, ios, ext_android_cb)
metrolyrics.com###videoContainer

**Review by BlazDT — APPROVED** [review]

**Review by BlazDT — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**